mods/gen4/random-teams.js

```diff
@@ -594,14 +596,18 @@ class RandomGen4Teams extends RandomGen5Teams {
 			item = 'Light Clay';
 		} else if (template.species === 'Palkia' && !!counter['Dragon'] && !!counter['Water']) {
 			item = 'Lustrous Orb';
+		} else if (ability === 'Serene Grace' && (hasMove['bodyslam'] || hasMove['headbutt'] || hasMove['ironhead'])) {
+			item = 'Leftovers';
+		} else if (ability === 'Guts') {
+			item = 'Toxic Orb';
```

Comment:
Does this apply to anything the higher priority check doesn't cover?

Reply:
Yeah, Heracross and Hariyama sets that don't have Sub but aren't eligible for Choice items so they currently get Life Orb. (SD + 3 attacks for Heracross, Fake Out + 3 attacks for Hariyama)

Reply:
Is that bad though? I thought mons ran Guts even without necessarily a Toxic/Flame Orb, if they don't have a better ability. Like AssVest Conk in previous generations. From my own experience, I think forcing the Orb on all Guts mons isn't the best idea. I don't even like Flame Orb Conk in gen 7 randoms; I just haven't gotten around to changing it.

Reply:
This check only ends up affecting cases where they would have had Life Orb. I think Toxic Orb is a bit better than Life Orb for those cases, since it's a bigger boost and the recoil ends up being about the same over time, plus it blocks sleep/para. I agree that they shouldn't always be forced to have an Orb, that's why I made this a separate lower priority check so that Heracross will still get Lefties if it has Sub and Band/Scarf if it's 4 attacks, and same for Guts Hariyama without Fake Out.

If you're still not convinced though I'm ok with removing this since it's not really a big difference.

Reply:
Fair enough.